packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -0,0 +1,429 @@
+        {props.children}
+      </RACAccordionItem>
+    </Provider>
+  );
+}
+
+/**
+ * A accordion item is a collapsible section of content. It is composed of a a header with a heading and trigger button, and a panel that contains the content.
+ */
+let _AccordionItem = /*#__PURE__*/ (forwardRef as forwardRefType)(AccordionItem);
+export {_AccordionItem as AccordionItem};
+
+export interface AccordionHeaderProps extends UnsafeStyles, DOMProps {
+  /** The heading level of the accordion header.
+   * 
+   * @default 3
+   */
+  level?: number,
+  children: React.ReactNode
+}
+
+const headingStyle = style({
+  margin: 0
+});
+
+const buttonStyles = style({
+  ...focusRing(),
+  outlineOffset: -2,
+  font: 'heading',
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  },
+  fontWeight: 'bold',
+  fontSize: {
+    size: {
+      S: 'ui-lg',
+      M: 'ui-xl',
+      L: 'ui-3xl',
+      XL: 'ui-5xl'
+    }
+  },
+  lineHeight: 'ui',
+  display: 'flex',
+  alignItems: 'center',
+  paddingX: {
+    size: {
+      S: sizeValue(7),
+      M: sizeValue(11),
+      L: sizeValue(14),
+      XL: sizeValue(17)
+    }
+  },
+  paddingTop: {
+    size: {
+      S: {
+        density: {
+          compact: sizeValue(2),
+          regular: sizeValue(5),
+          spacious: sizeValue(9)
+        }
+      },
+      M: {
+        density: {
+          compact: sizeValue(4),
+          regular: sizeValue(8),
+          spacious: sizeValue(12)
+        }
+      },
+      L: {
+        density: {
+          compact: sizeValue(4),
+          regular: sizeValue(9),
+          spacious: sizeValue(12)
+        }
+      },
+      XL: {
+        density: {
+          compact: sizeValue(5),
+          regular: sizeValue(9),
+          spacious: sizeValue(13)
+        }
+      }
+    }
+  },
+  paddingBottom: {
+    size: {
+      S: {
+        density: {
+          compact: sizeValue(2),
+          regular: sizeValue(7),
+          spacious: sizeValue(11)
+        }
+      },
+      M: {
+        density: {
+          compact: sizeValue(5),
+          regular: sizeValue(9),
+          spacious: sizeValue(13)
+        }
+      },
+      L: {
+        density: {
+          compact: sizeValue(8),
+          regular: sizeValue(11),
+          spacious: sizeValue(16)
+        }
+      },
+      XL: {
+        density: {
+          compact: sizeValue(8),
+          regular: sizeValue(12),
+          spacious: sizeValue(16)
+        }
+      }
+    }
+  },
+  gap: {
+    size: {
+      S: sizeValue(7),
+      M: sizeValue(11),
+      L: sizeValue(14),
+      XL: sizeValue(17)
+    }
+  },
+  height: 'full',
+  minHeight: 'control',
+  width: 'full',
+  backgroundColor: {
+    default: 'transparent',
+    isFocusVisible: 'transparent-black-100',
+    isHovered: 'transparent-black-100'
+  },
+  borderWidth: 0,
+  borderRadius: {
+    // Only rounded for keyboard focus and quiet hover.
+    default: 'none',
+    size: {
+      S: {
+        isFocusVisible: '[7px]',
```

Comment:
these odd values seem wrong, have you brought up with Spectrum?

Reply:
I haven't. I think they just wanted the smallest difference possible between sizes.

<img width="551" alt="Screenshot 2024-09-16 at 7 04 59 PM" src="https://github.com/user-attachments/assets/1ca2b890-d4dd-4109-a1c5-50c4dda613c1">